src/common/FlexGetEntry/index.js

```diff
@@ -0,0 +1,112 @@
+import { removeDupesIgnoreCase } from 'utils';
```

Comment:
I would move this file into the directory with the redux things. Still thinking about whether this is the best way to do this.